README.rst

```diff
@@ -33,23 +33,23 @@ Features
 * Pause/resume
 * Gzip support
 * Custom delimiters
-* parallel processing
+* Parallel processing
 
 
-Running batch_scoring
----------------------
+Running the batch_scoring or batch_scoring_sse scripts
+------------------------------------------------------
 
 You can execute the ``batch_scoring`` or ``batch_scoring_sse`` command from the command line or
-you can pass parameters to a ``batch_scoring`` or ``batch_scoring_sse`` script from .ini file. Place
-the file in your home directory or the directory from which you are running the ``batch_scoring``
+you can pass parameters to a ``batch_scoring`` or ``batch_scoring_sse`` script from the .ini file.
+Place the .ini file in your home directory or the directory from which you are running the ``batch_scoring``
 or ``batch_scoring_sse`` command. Use the syntax and arguments below to define the parameters.
 Note that if you run the script and also execute via the command line, the command line parameters take priority.
 
 The following table describes the syntax conventions; the syntax for running the script follows the table.
-Please note that we supply to different scripts:
+DataRobot supplies two scripts, each for a different application. Use:
 
-- ``batch_scoring`` is used to score over ORM and dedicated prediction instances.
-- ``batch_scoring_sse`` is used to score over standalone prediction instances.  If you not familiar with the instance types please contact your support.
+- ``batch_scoring`` to score on dedicated prediction instances.
+- ``batch_scoring_sse`` is used to score on standalone prediction instances. If you are unsure of your instance type, contact `DataRobot Support <https://support.datarobot.com/hc/en-us>`_.
```

Comment:
Is this formatted as a link? [DataRobot Support](https://support.datarobot.com/hc/en-us)?